h2o-core/src/main/java/water/H2ONode.java

```diff
@@ -41,7 +39,8 @@
     
   transient private short _timestamp;
   transient private boolean _removed_from_cloud;
-  transient private volatile boolean _accessed_local_dkv; // Did this remote node ever accessed the local portion of DKV?   
+  transient private boolean _accessed_local_dkv; // Did this remote node ever accessed the local portion of DKV?
```

Comment:
the volatile modifier should be preserved, why was it removed?

Reply:
Hm this is weird, must have been done by a mistake. Will put back.